client/securedrop_client/logic.py

```diff
@@ -1019,21 +1022,22 @@ def on_delete_source_failure(self, e: Exception) -> None:
             self.source_deletion_failed.emit(e.source_uuid)
 
     @login_required
-    def delete_source(self, source: db.Source) -> None:
+    def delete_sources(self, sources: set[db.Source]) -> None:
```

Comment:
I'm happy to see that this generalization worked out!

Reply:
That's the politest way I've ever heard of for a request that I fix a thing. Thank you.